lib/src/requests.dart

```diff
@@ -26,11 +26,11 @@ class Response {
 
   bool get success => !hasError;
 
-  Uri get url => _rawResponse.request.url;
+  Uri get url => _rawResponse.request!.url;
```

Comment:
_rawResponse is http.Response object, of which request is nullable. This may cause an assertion. I suggest making Uri? instead.